opts/parse.go

```diff
@@ -10,10 +10,10 @@ import (
 
 // ReadKVStrings reads a file of line terminated key=value pairs, and overrides any keys
 // present in the file with additional pairs specified in the override parameter
-func ReadKVStrings(files []string, override []string) ([]string, error) {
+func ReadKVStrings(files []string, override []string, emptyFn func(string) string) ([]string, error) {
```

Comment:
A couple of lines below, this still refers to `environment variables`.

Perhaps we should (for now)

- un-export `ParseKeyValueFile`
- have two wrappers; one variation of `ReadKVString` for labels, and one for environment variables
- those wrappers don't need the `emptyFn` as an argument (while nice, we don't really need that functionality at this point)

Reply:
😝 I can take care of that 👼